File: C++/ЕГЭ2021;2022/24/Решение заданий. Поляков/2558.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main() {
	string s;
	ifstream f("24-s1.txt");

	int count = 0;

	while (f >> s) {
		
		for (int i = 1; i < s.length(); i++) {
			if ((s[i - 1] == 'A') && (s[i + 1] == 'R')) {
				count++;
				break;
			}
		}

	}
	cout << count;
	f.close();

	return 0;

}

File: Python/ЕГЭ2021;2022/24/Решение заданий. Поляков/2558.py
f = open('24-s1.txt')


count = 0

while True:
    s = f.readline()

    if not s:
        break

    for i in range(1, len(s)-1):
        if s[i-1] == 'A' and s[i+1] == 'R':
            count += 1
            break

print(count)
